docs/configuring-os2loop.md

```diff
@@ -24,10 +24,20 @@ In order to make the media library work, go to
 media categories for the site.
 
 Go to `/admin/structure/taxonomy` and make sure that the Profession and Subject
-taxonomies containt the terms needed for the content. Users can dynamically
+taxonomies contain the terms needed for the content. Users can dynamically
 create new terms in the Tag taxonomy, but any default terms can be added if
 needed.
 
+All taxonomy terms are added and edited on '/admin/structure/taxonomy'
+
+Category: Is used for overall grouping of content e.g. "Vejledning" or "Instruks".
+Subject: Is used for grouping content related to similar themes (subjects)
+e.g. "Support" or "Administration".
+Tags: Is used as inspirational words for similar content
+e.g. "Ernæringsindsats" or "Hjælpemidler".
+As opposed to the other taxonomy terms these can also be defined by
+end users when submitting questions and/or answers.
```

Comment:
The list of taxonomies will be shown as a single paragraph (cf. https://github.com/os2loop/os2loop/blob/949194075f85fcdf56005968227cc9dacd24d9d5/docs/configuring-os2loop.md#taxonomies). Consider using a list to make it easier to read (https://www.markdownguide.org/basic-syntax/#lists-1).

Reply:
Made a list of taxonomy terms